riverdriver/riverdatabasesql/river_database_sql_driver_test.go

```diff
@@ -78,6 +78,8 @@ func TestReplaceNamed(t *testing.T) {
 		{Desc: "SliceUint64", ExpectedSQL: "SELECT ARRAY[123,124]", InputSQL: "SELECT @slice_uint64", InputArgs: map[string]any{"slice_uint64": []uint64{123, 124}}},
 	}
 	for _, tt := range testCases {
+		tt := tt
```

Comment:
Something I hadn't realized is that golangci-lint is actually reading the Go version from `go.mod`. So as we reset the min version down to 1.21, the lint requiring that loop vars are captured kicked back in.